Uplift/Views/GymDetail/GymDetailWeekView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  GymDetailCalendarView.swift
+//  Uplift
+//
+//  The header calendar used to select which content to display based on day of the week
+//
+//  Created by Phillip OReggio on 10/2/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymDetailWeekView: UIView {
+
+    // MARK: - Public
+    var selectedDay: WeekDay?
+
+    // MARK: - Display
+    private var weekdayCollectionView: UICollectionView!
+    private let daysSize = CGSize(width: 24, height: 24)
+    private let interitemSpace: CGFloat = 19
+
+    // MARK: - Info
+    private var days: [WeekDay] = [.monday, .tuesday, .wednesday, .thursday, .friday, .saturday, .sunday]
```

Comment:
same as before, keep order consistent and start with sunday

Reply:
I use that order to index into the correct day without any mod